src/Integration.Vsix/SonarLintTagger/IssueMarker.cs

```diff
@@ -30,20 +31,36 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// <remarks>
     /// See the README.md in this folder for more information
     /// </remarks>
-    internal class IssueMarker
+    internal class IssueMarker : IFilterableIssue
     {
         public IAnalysisIssue Issue { get; }
         public SnapshotSpan Span { get; }
+        public string WholeLineText { get; set; }
+        public string LineHash { get; set; }
```

Comment:
I've added these two properties which are the ones required by `IFilterableIssue`.